packages/signer/src/api/ws.sagas.ts

```diff
@@ -146,6 +149,25 @@ const getOriginHost = (request: IncomingMessage) => {
   }
 };
 
+/**
+ * Verifies the JSON-RPC request nonce based on the ID and public key. If the nonce is valid, the
```

Comment:
Perhaps add something about: "we expect the JSON RPC ids to be nonces and increment for each request..."?